account/models.py

```diff
@@ -1,3 +1,15 @@
 from django.db import models
 
-# Create your models here.
+class Account(models.Model):
+    name           =   models.CharField(max_length=50)
+    email          =   models.EmailField(max_length=50)
+    password       =   models.CharField(max_length=100)
+    language       =   models.CharField(max_length=50, null=True)
+    country        =   models.CharField(max_length=50, null=True)
+    birthday       =   models.IntegerField(default=0,  null=True)
+    phone          =   models.IntegerField(default=0)
```

Comment:
```suggestion
    phone          =   models.CharField(default=0)
```
핸드폰 번호는 정수가 아닙니다. 숫자처럼 보이긴 하지만 사실 '정수'로서의 역할을 하지 않죠. 정수라면 번호 제일 앞의 `0`이 무시될 수도 있습니다 문자열로 관리해주세요 :)